### DISCUSSION TITLE
Module Style Components not found

### DISCUSSION THREAD

**Discussion by jgraffin** [rigelglen/react-paginated-list#14] [Troubleshooting] [answered]
Hello!

I tried to  use this component here, but I'm stuck in an import component dependency.
Can you help on how to solve this?
Thanks!

![image](https://user-images.githubusercontent.com/7095196/125114652-9dc1d280-e0c0-11eb-8d46-8d74d9eb239e.png)

**Comment by rigelglen** [comment, marked as answer]
Hi, 

`styled-components` is a peer dependency and you need to install it

Install it with 

`npm i styled-components`
